lib/icons/icon-tables.js

```diff
@@ -69,11 +69,13 @@ class IconTables{
 			: [cache.fileName,      this.fileIcons.byName];
 		const cached = cachedIcons.get(name);
 		if(cached) return cached;
-		for(const icon of icons)
+		for (let i = 0, l = icons.length; i < l; ++i) {
```

Comment:
Please remove the bracket spacing from each loop:

~~~diff
-for (let i = 0, l = icons.length; i < l; ++i) {
+for(let i = 0, l = icons.length; i < l; ++i){
~~~